client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriver.java

```diff
@@ -743,17 +743,19 @@ public void testSetRole()
     {
         try (TrinoConnection connection = createConnection(TEST_CATALOG, "tiny").unwrap(TrinoConnection.class)) {
             try (Statement statement = connection.createStatement()) {
-                statement.executeUpdate("SET ROLE ALL");
+                statement.executeUpdate("SET ROLE ALL IN " + TEST_CATALOG);
             }
             assertEquals(connection.getRoles(), ImmutableMap.of(TEST_CATALOG, new ClientSelectedRole(ClientSelectedRole.Type.ALL, Optional.empty())));
             try (Statement statement = connection.createStatement()) {
-                statement.executeUpdate("SET ROLE NONE");
+                statement.executeUpdate("SET ROLE NONE IN " + TEST_CATALOG);
             }
             assertEquals(connection.getRoles(), ImmutableMap.of(TEST_CATALOG, new ClientSelectedRole(ClientSelectedRole.Type.NONE, Optional.empty())));
-            try (Statement statement = connection.createStatement()) {
-                statement.executeUpdate("SET ROLE bar");
-            }
-            assertEquals(connection.getRoles(), ImmutableMap.of(TEST_CATALOG, new ClientSelectedRole(ClientSelectedRole.Type.ROLE, Optional.of("bar"))));
+
+            // Only hive connector supports roles
+            // try (Statement statement = connection.createStatement()) {
+            //     statement.executeUpdate("SET ROLE bar IN " + TEST_CATALOG);
+            // }
+            // assertEquals(connection.getRoles(), ImmutableMap.of(TEST_CATALOG, new ClientSelectedRole(ClientSelectedRole.Type.ROLE, Optional.of("bar"))));
```

Comment:
Do you have a test for that in Hive? If so let's drop it (in separate commit). Or maybe it is already covered with `TestSetRoleTask`?

Reply:
There are a bunch of tests for this.  If you are curious search the code base for `"set role`

Reply:
Then please just drop the code then, instead of commenting.